examples/basic/axes/twin_axes.py

```diff
@@ -11,23 +11,26 @@
 
 from bokeh.models import LinearAxis, Range1d
 from bokeh.plotting import figure, show
+from bokeh.palettes import Sunset6
 
 x = arange(-2*pi, 2*pi, 0.2)
 y = sin(x)
 y2 = linspace(0, 100, len(x))
 
+blueshade, redshade = Sunset6[2], Sunset6[5]
+
 p = figure(x_range=(-2*pi, 2*pi), y_range=(-1, 1))
 p.background_fill_color = "#fafafa"
 
-p.scatter(x, y, color="crimson", size=8)
-p.yaxis.axis_label = "red circles"
-p.yaxis.axis_label_text_color ="crimson"
+p.scatter(x, y, line_color="black", fill_color=blueshade, size=12)
+p.yaxis.axis_label = "light blue circles"
+p.yaxis.axis_label_text_color ="blue"
```

Comment:
this can be set to the exact same color `blueshade` (or `blue` with the rename I suggested) so that it will match the glyph color exactly